redbot/core/data_manager.py

```diff
@@ -9,21 +9,18 @@
 from typing import Any, Dict
 
 import platformdirs
-from discord.utils import deprecated
 
 from . import commands
-from .cli import ExitCodes
+from ._cli import ExitCodes
 
-__all__ = [
-    "create_temp_config",
-    "load_basic_configuration",
+__all__ = (
     "cog_data_path",
-    "core_data_path",
```

Comment:
Not certain if this should be private. At least one existing 3rd party cog uses it (Toby's `errorlogs`), and I'm not sure of the benefits and drawbacks of keeping it public.

Reply:
The idea behind it was that 3rd-party cogs should be using `cog_data_path()` so there should be no reason to use `core_data_path()` but the use case of `core_data_path() / "logs"` is not a bad one. Of course there's the `data_path() / "core/logs"` but I don't see a reason to have API users jump through these hoops. Gonna revert this removal.